mongodbatlas/search.go

```diff
@@ -281,7 +283,8 @@ type IndexField struct {
 // SearchAnalyzer custom analyzer definition.
 type SearchAnalyzer struct {
 	BaseAnalyzer     string   `json:"baseAnalyzer"`
-	MaxTokenLength   *float64 `json:"maxTokenLength,omitempty"`
+	MaxTokenLength   *int     `json:"maxTokenLength,omitempty"`
```

Comment:
any reason for this change? it would be better if we don't but not a blocker

Reply:
According to the API documentation for search kndexes and analyzers, this property is an int type.

And that makes more sense (how you can limit a token using a float value?)

What do you think?

Reply:
not a blocker like I said but it's a breaking change so trying to avoid it if we can, since for json all number fields are technically float, the previous implementation was valid (even if wrong) but feel free to merge

Reply:
ok, It's good to know, so this time I'll merge that change

Thanks for your comment Gustavo